app/lib/common/widgets/spaces/space_hierarchy_card.dart

```diff
@@ -160,8 +165,17 @@ class SpaceHierarchyCard extends ConsumerWidget {
     final roomId = space.roomIdStr();
     final profile = ref.watch(spaceHierarchyProfileProvider(space));
     final topic = space.topic();
-    final Widget? subtitle =
-        topic != null && topic.isNotEmpty ? Text(topic) : null;
+    final Widget? subtitle = topic != null && topic.isNotEmpty
+        ? ReadMoreText(
```

Comment:
I'd suggest using [expandable_text](https://pub.dev/packages/expandable_text), that's currently existing as utility package for handling cases like these.

Reply:
Oh I see. I will reuse the existing library here.  Thanks for letting me know about the same. :)